lib/collections/addon/index/route.ts

```diff
@@ -1,11 +0,0 @@
-import Store from '@ember-data/store';
-import Route from '@ember/routing/route';
-import { inject as service } from '@ember/service';
-
-export default class Index extends Route {
-    @service store!: Store;
-
-    model() {
-        return this.store.findAll('collection-provider');
-    }
-}
```

Comment:
@brianpilati My guess is that this is going to have to go back in to make the index page redirect automatically to the discover page. That or `lib/collections/addon/provider/index/route.ts` will have to be re-worked to not try to import from this file.